skeleton_scary_game.py

```diff
@@ -0,0 +1,151 @@
+import time
+import random
+
+# This is where the actual game will run from
+class game_engine(object):
+
+    # This is the constructer method, this is what will be passed to the play() for the game to run
+    def __init__(self,demon,item,father,lore):
+        self.demon = demon
+        self.item = item
+        self.father = father
+        self.paths = ["Path 1", "Path 2", "Path 3"]
+        self.lore = lore    
+    
+    # This is the actual game where everything will run
+    def play(self,):
+
+        # how many rounds you have survived,
+        loop_count = 0
+
+        # Loop until 7 rounds have been reach and you win, the game is on a finite condition.
+        while loop_count < 7:
+
+            # The path the demon will be on - randomizes every round
+            demon_path = random.choice(self.paths)
+            
+
+            # Randomly assign the item path
+            item_paths = [path for path in self.paths if path != demon_path]
```

Comment:
you should change item_paths to `possible_item_paths` so it's easier to understand. Alternatively you could just put the code that creates the list you assign to `item_paths` into `random.choice()`.